Tools/CISupport/ews-build/steps.py

```diff
@@ -158,6 +158,21 @@ def get_pr_json(self, pr_number, repository_url=None):
             return None
         return pr_json
 
+    def get_pr_patch(self, pr_number, repository_url=None):
```

Comment:
Please check if we need this or we have this information pre-populated by default buildbot _get_pr_files() method  (being modified in https://github.com/WebKit/WebKit/pull/83/files).

Reply:
I do think this is worth considering, but it requires re-factoring `AnalyzeChange` to return a list of changed files instead of a patch.

Reply:
That would be pretty straight-forward, just need to return a string from this get_pr_patch() method (maybe something like: `return ', '.format(file_list)`, and rest should work fine.

If we can save this network call, that would be much better and faster.

Reply:
If we can let buildbot fetch the info on per-PR basis as proposed in https://github.com/WebKit/WebKit/pull/79, and then if we can skip this network call here, that would be nice.
Since it would be one network call per PR (if buildbot does it for us), rather than one network call per build (in AnaylzePatch as proposed in this change). And one PR can have 30+ builds.

Reply:
I'll look into this.